weblate/templates/manage-access.html

```diff
@@ -52,7 +52,7 @@
   </tbody>
   </table>
   <div class="panel-body">
-  <p class="helptext">{% trans "The user will be removed from the project once all user permissions are removed." %}</p>
+  <p class="helptext">{% trans "The user will be removed from the project once all user permissions are." %}</p>
```

Comment:
Are you sure with this one? It sounds weird to me.

Reply:
I think if I have to argue that I loose.